ta_services/secure_key_services/src/sks_helpers.c

```diff
@@ -0,0 +1,512 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2017-2018, Linaro Limited
+ */
+
+#include <sks_ta.h>
+#include <string.h>
+
+#include "sks_helpers.h"
+
+struct string_id {
+	uint32_t id;
+	const char *string;
+};
+
+struct attr_size {
+	uint32_t id;
+	uint32_t size;
+	const char *string;
+};
+
+#if CFG_TEE_TA_LOG_LEVEL > 0
+#define SKS_ID_SZ(_id, _size)	{ .id = _id, .string = #_id, .size = _size }
+#define SKS_ID(_id)		{ .id = _id, .string = #_id }
+#else
+#define SKS_ID_SZ(_id, _size)	{ .id = _id, .size = _size }
+#endif
+
+static const struct attr_size attr_ids[] = {
+	SKS_ID_SZ(SKS_CLASS,		4),
+	SKS_ID_SZ(SKS_TYPE,		4),
+	SKS_ID_SZ(SKS_VALUE,		0),
+	SKS_ID_SZ(SKS_VALUE_LEN,	4),
+	SKS_ID_SZ(SKS_WRAP_ATTRIBS,	0),
+	SKS_ID_SZ(SKS_UNWRAP_ATTRIBS,	0),
+	SKS_ID_SZ(SKS_DERIVE_ATTRIBS,	0),
+	SKS_ID_SZ(SKS_ACTIVATION_DATE,	4),
+	SKS_ID_SZ(SKS_REVOKATION_DATE,	4),
+	SKS_ID_SZ(SKS_OBJECT_ID,	0),
+	SKS_ID_SZ(SKS_APPLICATION_ID,	0),
+	SKS_ID_SZ(SKS_PROCESSING_ID,	4),
+	SKS_ID_SZ(SKS_KEY_ID,		0),
+	SKS_ID_SZ(SKS_ALLOWED_PROCESSINGS, 0),
```

Comment:
`0),` is not aligned properly with the previous entry. Personally I would not try to align at all.

Reply:
ok